helpers/lib/fonts.js

```diff
@@ -51,8 +51,8 @@ const fontProviders = {
             return collection;
         },
 
-        buildLink: function(fonts) {
-            return '<link href="https://fonts.googleapis.com/css?family=' + fonts.join('|') + '" rel="stylesheet">';
+        buildLink: function(fonts, fontDisplay) {
+            return `<link href="https://fonts.googleapis.com/css?family=${fonts.join('|')}&display=${fontDisplay ? fontDisplay : 'swap'}" rel="stylesheet">`;
```

Comment:
You might want to do a check to make sure the value is valid, i.e.: `auto | block | swap | fallback | optional`. 🍺

Reply:
Thanks, added that check and added unit tests for it. Made it go to default if invalid.